### DISCUSSION TITLE
capture="user" not working in Chrome Browser

### DISCUSSION THREAD

**Discussion by minhtam1004** [mdn/mdn-community#243] [Browser compatibility data]
I have this issue: capture="user" not working in Chrome Browser
Please help me :(((

**Comment by pepelsbey** [comment]
Thank you for starting the discussion. I’m closing this now due to inactivity, but if you think that’s a mistake, feel free to reply. You’re also welcome to join the [MDN Discord](https://developer.mozilla.org/discord) if you’d prefer to chat with us more about it in sync.
